### ISSUE TITLE
Haprxy 

### ISSUE THREAD

**Issue by issaouib** [haproxytechblog/ingress-controller-external-example#3]
Hello
how to configure haproxy for multi master cluster with haproxy-ingress-controller on the same time, I did not deactivate the haproxy service as you indicated in the tutorial because I use it as HA for multi controleplane. the HA service is running but HA-ingress-controller is not.
for the Bird.conf configuration I assigned the 3 controlplanes.
# controlplane1
protocol bgp {
  	local 192.168.50.21 as 65000;
	neighbor 192.168.50.22 as 65000;
	direct;
    import filter {
      if ( net ~ [ 172.16.0.0/16{26,26} ] ) then accept;
    };
    export none;
}
# controlplane2
protocol bgp {
  	local 192.168.50.23 as 65000;
	neighbor 192.168.50.22 as 65000;
	direct;
    import filter {
      if ( net ~ [ 172.16.0.0/16{26,26} ] ) then accept;
    };
    export none;
}
# controlplane3
protocol bgp {
  	local 192.168.50.21 as 65000;
	neighbor 192.168.50.24 as 65000;
	direct;
    import filter {
      if ( net ~ [ 172.16.0.0/16{26,26} ] ) then accept;
    };
    export none;
}
# worker1
protocol bgp {
  	local 192.168.50.21 as 65000;
	neighbor 192.168.50.25 as 65000;
	direct;
    import filter {
      if ( net ~ [ 172.16.0.0/16{26,26} ] ) then accept;
    };
    export none;
}

# worker2
protocol bgp {
  	local 192.168.50.21 as 65000;
	neighbor 192.168.50.26 as 65000;
	direct;
    import filter {
      if ( net ~ [ 172.16.0.0/16{26,26} ] ) then accept;
    };
    export none;
}

**Comment by NickMRamirez** [comment]
@issaouib I suspect you will not be able to use the same HAProxy service for both use cases, because the ingress controller wants to own / manage HAProxy. I think you would need a separate load balancer for balancing the master nodes. I fear that your configuration would be overwritten otherwise. Have you experienced otherwise?

**Comment by vhelke** [comment]
I'm also interested in this. Has anyone found this possible?

**Comment by NumenDivinum** [comment]
@vhelke have you tried defining kube-api for cluster as ingress resource with ssl-passthrough?

**Comment by vhelke** [comment]
No, but that is an interesting idea. However, we have settled for two pairs of haproxies for the time being.
